src/applications/dto/create-application.dto.ts

```
@@ -1,4 +1,5 @@
 export class CreateApplicationDto {
+     jobId: string;
```

ESLint

[error] Delete `···`

(prettier/prettier)